service/frontend/workflowHandler.go

```diff
@@ -2176,27 +1827,47 @@ func (wh *WorkflowHandler) GetWorkflowExecutionHistory(
 		token.IsWorkflowRunning = isWorkflowRunning
 		token.PersistenceToken = nil
 	}
+	rawHistoryQueryEnabled := wh.config.EnableRawHistoryQuery() && wh.config.EnableRawHistoryQueryByDomain(getRequest.GetDomain())
```

Comment:
see comment above - just ByDomain config is sufficient

Reply:
> see comment above - just ByDomain config is sufficient

done